packages/saber/lib/plugins/extend-node-api.js

```diff
@@ -86,7 +88,7 @@ exports.apply = api => {
           )
           // Because you might also update webpack config in saber-node.js
           // Which we can't (?) automatically reload
-          log.warn(`You probably need to restart the server.`)
+          notifier.notify('You probably need to restart the server.')
```

Comment:
we should still output warning in console

Reply:
Oops. sorry.

Reply:
We should set `Saber` as the title